### PR TITLE
Debug drone

### PR DESCRIPTION
**Pull request by gileshinchcliff**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -33,20 +33,32 @@ pipeline:
...
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - TAGS=platform:sysdig-dev,environment:SYSDIG-DEV
+      - COLLECTOR=sysdig-collector-dev.ops.digital.homeoffice.gov.uk.
+      - NAME=sysdig-agent-dev
+    commands:
+      - export SYSDIG_ACCESS_TOKEN=${DEV_SYSDIG_ACCESS_TOKEN}
+      - ./kdaemon -n platform-services -x name=sysdig-agent-dev -s https://kube-dev.dsp.notprod.homeoffice.gov.uk -r 10 -t ${KUBE_TOKEN_DEV} -f k8s/agent-daemonset.yaml
+    when:
+      branch: master
+      event: push
```
Comment (marcinc) [inline .drone.yml]: Should this happen on `tag` like other deployments below?

Comment (gileshinchcliff) [inline .drone.yml]: no, the point is that, this deploy is sending metric to a dev instance of sysdig. So this is like a sandbox.

File: .drone.yml [hunk 2]
```diff
@@ -33,20 +33,32 @@ pipeline:
       - docker tag sysdig-agent quay.io/ukhomeofficedigital/sysdig-agent:${DRONE_COMMIT_SHA}
       - docker push quay.io/ukhomeofficedigital/sysdig-agent:${DRONE_COMMIT_SHA}
     when:
-      branch: master
+      branch: debug-drone
```
Comment (gambol99) [inline .drone.yml]: debug-drone??

**Comment by gambol99** [comment]
LGTM

**Pull request merged**